docs/src/docs/reports/assertions.adoc

```diff
@@ -0,0 +1,24 @@
+= More on Assertions
+
+== Collected assertions
```

Comment:
Maybe add a link to this part in <<#_collected_assertions, Collected assertions>> and <<#_optional__collected_assertions, Optional assertions>> as well so someone reading how to use the functionality can immediatly check how it will look in the report

Reply:
There are already links to thease chapters.